app/docs/dev/policies/meshaccesslog.md

```diff
@@ -0,0 +1,287 @@
+    - key: "bytes_received"
+      value: "%BYTES_RECEIVED%"
+```
+
+Example output:
+
+```json
+{
+  "start_time": "2016-04-15T20:17:00.310Z",
+  "bytes_received": "154"
+}
+```
+
+<details>
+  <summary>TCP configuration with default fields:</summary>
+
+  <div markdown="1">
+```yaml
+format:
+  json:
+    - key: "start_time"
+      value: "%START_TIME%"
+    - key: "response_flags"
+      value: "%RESPONSE_FLAGS%"
+    - key: "kuma_mesh"
+      value: "%KUMA_MESH%"
+    - key: "kuma_source_address_without_port"
+      value: "%KUMA_SOURCE_ADDRESS_WITHOUT_PORT%"
+    - key: "kuma_source_service"
+      value: "%KUMA_SOURCE_SERVICE%"
+    - key: "upstream_host"
+      value: "%UPSTREAM_HOST%"
+    - key: "kuma_destination_service"
+      value: "%KUMA_DESTINATION_SERVICE%"
+    - key: "duration_ms"
+      value: "%DURATION%"
+    - key: "bytes_sent"
+      value: "%BYTES_SENT%"
+    - key: "bytes_received"
+      value: "%BYTES_RECEIVED%"
+```
+</div>
+
+</details>
+
+<details>
+  <summary>HTTP configuration with default fields:</summary>
+
+<div markdown="1">
+```yaml
+format:
+  json:
+    - key: "start_time"
+      value: "%START_TIME%"
+    - key: "kuma_mesh"
+      value: "%KUMA_MESH%"
+    - key: 'method'
+      value: '"%REQ(:METHOD)%'
+    - key: "path"
+      value: "%REQ(X-ENVOY-ORIGINAL-PATH?:PATH)%"
+    - key: 'protocol'
+      value: '%PROTOCOL%'
+    - key: "response_code"
+      value: "%RESPONSE_CODE%"
+    - key: "response_flags"
+      value: "%RESPONSE_FLAGS%"
+    - key: "bytes_received"
+      value: "%BYTES_RECEIVED%"
+    - key: "bytes_sent"
+      value: "%BYTES_SENT%"
+    - key: "duration_ms"
+      value: "%DURATION%"
+    - key: "upstream_service_time"
+      value: "%RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)%"
+    - key: 'x_forwarded_for'
+      value: '"%REQ(X-FORWARDED-FOR)%"'
+    - key: 'user_agent'
+      value: '"%REQ(USER-AGENT)%"'
+    - key: 'request_id'
+      value: '"%REQ(X-REQUEST-ID)%"'
+    - key: 'authority'
+      value: '"%REQ(:AUTHORITY)%"'
+    - key: "kuma_source_service"
+      value: "%KUMA_SOURCE_SERVICE%"
+    - key: "kuma_destination_service"
+      value: "%KUMA_DESTINATION_SERVICE%"
+    - key: "kuma_source_address_without_port"
+      value: "%KUMA_SOURCE_ADDRESS_WITHOUT_PORT%"
+    - key: "upstream_host"
+      value: "%UPSTREAM_HOST%"
+```
+</div>
+
+</details>
+
+
+### Backends
+
+#### TCP
+
+You can configure a TCP backend with an address:
+
+```yaml
+backends:
+  - tcp:
+      address: 127.0.0.1:5000
+```
+
+#### File
+
+You can configure a file backend with a path:
+
+```yaml
+backends:
+  - file:
+      path: /tmp/access.log
+```
+
+## Examples
+
+{% tabs meshaccesslog useUrlFragment=false %}
+{% tab meshaccesslog Kubernetes %}
+
+Full example:
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshAccessLog
+metadata:
+  name: default
+  namespace: kuma-system
+  labels:
+    kuma.io/mesh: default # optional, defaults to `default` if unset
+spec:
+  targetRef:
+    kind: Mesh
+  default:
```

Comment:
These don't seem valid they don't have to and from

Reply:
We should also probably explain the point of `to` and `from` and instead of having just Full examples and simple examples just have real examples.

.e.g: logging traffic coming from a single service on a specific dataplane, logging to multiple backends, log all incoming traffic, all outgoing traffic.

Reply:
To me a full example is something that shows all configuration options available. The best is to combine both of them, have some functional examples and a reference one showing all options (with optional ones).